joblib/testing.py

```diff
@@ -51,6 +68,24 @@ def assert_raises_regex(expected_exception, expected_regexp,
                                      expected_exception(expected_regexp))
 
 
+try:
```

Comment:
Maybe it should be a try on an import of assert_raises from unittest and if it fails (python 2.6), import assert_raises from  nose.tools ?